azure.yaml

```
@@ -0,0 +1,10 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
+
+name: functions-quickstart-typescript-azd
+metadata:
+  template: functions-quickstart-typescript-azd@0.0.2-beta
```

```suggestion
  template: functions-quickstart-typescript-azd@1.0.0-beta
```
I think we can flip to 1.0 beta or just 1.0.